pkg/operator/starter.go

```diff
@@ -234,6 +262,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	configInformers.Start(ctx.Done())
 	dynamicInformers.Start(ctx.Done())
 
+	go staticResourceController.Run(ctx, 1)
```

Comment:
if it will ever make it to library-go it would be nice if we could let the callers know that setting workers will not be respected.

Reply:
> if it will ever make it to library-go it would be nice if we could let the callers know that setting workers will not be respected.

sure